server/index.js

```diff
@@ -24,14 +25,16 @@ async function createServer () {
     }
   })
 
-  // Areas and offices change infrequently. Expire cache daily. Query _should_ take < 2 seconds, allows 3x before error.
+  // Areas and offices change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 3x before error.
   server.method('db.getAreaToOfficeMap', getAreaToOfficeMap, { cache: { expiresIn: 60 * 60 * 1000, generateTimeout: 6 * 1000 } })
-  // Organisations change infrequently. Expire cache daily. Query _should_ take < 2 seconds, allows 3x before error.
+  // Organisations change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 3x before error.
   server.method('db.getOrganisationList', getOrganisationList, { cache: { expiresIn: 60 * 60 * 1000, generateTimeout: 6 * 1000 } })
-  // Office locations change infrequently. Expire cache daily. Query _should_ take < 2 seconds, allows 3x before error.
+  // Office locations change infrequently. Expire cache hourly. Query _should_ take < 2 seconds, allows 3x before error.
   server.method('db.getStandardisedOfficeLocationMap', getStandardisedOfficeLocationMap, { cache: { expiresIn: 60 * 60 * 1000, generateTimeout: 6 * 1000 } })
   // Users are scheduled to refresh weekly. Expire cache daily. Query _should_ take ~10 seconds, allows 3x before error.
   server.method('db.getUsers', getUsers, { cache: { expiresAt: '00:05', generateTimeout: 30 * 1000 } })
+  // Notify status doesn't change frequently but is important to be up to date. Expire cache every 5 minutes. Query _should_ take < 2 seconds, allows 3x before error.
+  server.method('getNotifyStatusViewData', getNotifyStatusViewData, { cache: { expiresIn: 5 * 60 * 1000, generateTimeout: 6 * 1000 } })
```

Comment:
I did consider a prefix for consistency but couldn't think of a decent one and I didn't want it to be something like `misc`. With that in mind I decided to leave it off and with the current creation of the methods all being in this file figured it would be pretty easy to revisit that in the future, should it become more fitting to have a prefix.